cmd/reset.go

```diff
@@ -30,6 +31,13 @@ var resetCmd = &cobra.Command{
 		cloudProvider := viper.GetString("kubefirst.cloud-provider")
 
 		checksMap := viper.Get("kubefirst-checks")
+		if checksMap == nil {
+			in := `# Succesfully reset`
+			out, _ := glamour.Render(in, "dark")
+			fmt.Print(out)
+			progress.Progress.Quit()
+			return nil
+		}
```

Comment:
Down below there seems to be a switch on the type, which could have the type to be empty (`nil`).

Do we need to quit if we can't find it? Or perhaps we need to ignore it if it's not there but continue with the cleanup after?

Reply:
Strange, because right down below, it says "if the map is a string, and that string is empty, log a message but continue".

What makes the map being nil vs being empty different?

Reply:
if we have .kubefirst file present,then in that file we will have "kubefirst-checks" as one of the fields,which will be set to empty as "kubefirst-checks:""' and we run a reset cmd,however when we dont have that file present at all go thows nil error

Reply:
we can remove the else in case string ig